scripts/index-discovery.md

```diff
@@ -0,0 +1,54 @@
+# index-discovery script
+[Back to the script index](index.md)
+<!-- TOC -->
+* [index-discovery script](#index-discovery-script)
+  * [What it does?](#what-it-does)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters:](#parameters)
+  * [Conclusion/QuickGuide:](#conclusionquickguide)
+<!-- TOC -->
+## What it does?
+
+The index-discovery script interacts with the Discovery Solr search index in DSpace. Depending on the parameters passed,
```

Comment:
This should link back to the main Discovery documentation at https://wiki.lyrasis.org/display/DSDOC7x/Discovery